main.py

```diff
@@ -53,36 +53,44 @@
 # We use cookie on client site after user successfully logged in
 class Login:
 
-    # Check html and json files
+    # check file existence
     def _check_the_file(self, filename):
-        self.__filename = filename
-        _file = os.path.isfile(self.__filename)
-        if not _file:
-            print("Internal server error! " \
-                  "file not found:{}".format(self.__filename))
+        if not os.path.isfile(filename):
+            print("Internal server error! "
+                  "file not found:{}".format(filename))
             return False
+        return True
 
-        # Read html files
-        if self.__filename.endswith(".html"):
-            response = open(self.__filename).read()
-            return response
-
-        # Read json file
-        elif self.__filename.endswith(".json"):
-            configure_file = open(self.__filename)
-            load_data = json.load(configure_file)
-            if all(key in load_data for key in("USERNAME", "PASSWORD")):
-                return load_data
+    # read html files
+    # return False if requested file format is not .html
+    def _load_html_file(self, filename):
+        if not self._check_the_file(filename):
+            return False
+        if not filename.endswith(".html"):
+            print("{} is not html file".format(filename))
             return False
+        response = open(filename).read()
+        return response
```

Comment:
return open ..., No need to store in response variable for nothing. Why did you store it?